### PR TITLE
DealTime - product APIM training material 

### PR DESCRIPTION
**Pull request by wijith7**

### CODE REVIEW

**Review by nuwand — APPROVED** [review]

**Comment by madurangasiriwardena** [comment]
PR has several .DS_Store files. Please remove them.

File: Dealtime/distribution/pom.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
```
Comment (madurangasiriwardena) [inline Dealtime/distribution/pom.xml:2]: License header seems to be not correct.

File: Dealtime/distribution/pom.xml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+
+
+</project>
```
Comment (madurangasiriwardena) [inline Dealtime/distribution/pom.xml:85]: Add new line

File: Dealtime/distribution/src/main/assembly/bin.xml [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (madurangasiriwardena) [inline Dealtime/distribution/src/main/assembly/bin.xml:3]: License header is not correct. Correct all the places.

File: Dealtime/distribution/src/main/assembly/bin.xml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            <outputDirectory>project/Front-End/host/build</outputDirectory>
+        </fileSet>
+         <fileSet>
+            <directory>${basedir}/../html-files</directory>
+            <outputDirectory>project/Front-End/host/build</outputDirectory>
+        </fileSet>
+
+
+    </fileSets>
+
```
Comment (madurangasiriwardena) [inline Dealtime/distribution/src/main/assembly/bin.xml:77]: unnecessary new lines

File: Dealtime/distribution/pom.xml (hunk 1 above)
Comment (wijith7) [reply Dealtime/distribution/pom.xml:2]: fix with - https://github.com/wso2/product-apim-training/pull/2

File: Dealtime/distribution/pom.xml (hunk 2 above)
Comment (wijith7) [reply Dealtime/distribution/pom.xml:85]: fix - with https://github.com/wso2/product-apim-training/pull/2

File: Dealtime/distribution/src/main/assembly/bin.xml (hunk 3 above)
Comment (wijith7) [reply Dealtime/distribution/src/main/assembly/bin.xml:3]: fixed with - https://github.com/wso2/product-apim-training/pull/2

File: Dealtime/distribution/src/main/assembly/bin.xml (hunk 4 above)
Comment (wijith7) [reply Dealtime/distribution/src/main/assembly/bin.xml:77]: fixed with -https://github.com/wso2/product-apim-training/pull/2

**Pull request merged**